examples/vue-nuxt/package.json

```diff
@@ -10,7 +10,7 @@
     "generate": "NODE_PRESERVE_SYMLINKS=1 nuxt generate"
   },
   "dependencies": {
-    "@builder.io/sdk-vue": "0.0.1-44",
+    "@builder.io/sdk-vue": "^0.0.1-47",
```

Comment:
given that this is a v0 package and there will be a lot of change it might make sense to lock it to an exact version to guarantee that it continues to work